### PR TITLE
Bug fixed + Refact

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Forms/DoNotBreakAfterListEdit.cs [hunk 1]
```diff
@@ -208,16 +205,10 @@ private void listBoxNamesEtc_SelectedIndexChanged(object sender, EventArgs e)
             {
                 NoBreakAfterItem item = _noBreakAfterList[idx];
                 textBoxNoBreakAfter.Text = item.Text;
-                if (item.Regex != null)
```
Comment (ivandrofly) [inline src/Forms/DoNotBreakAfterListEdit.cs]: Bug found here :wink:

Comment (niksedk) [inline src/Forms/DoNotBreakAfterListEdit.cs]: cool :)
I've committed it (along with another bugfix) + correct tab order here: https://github.com/SubtitleEdit/subtitleedit/commit/2877320b76893edb9f523f1efa47359fe03adca7

File: src/Logic/SsaStyle.cs [hunk 2]
```diff
@@ -107,8 +107,7 @@ internal string ToRawSsa(string styleFormat)
                     sb.Append('0');
                 sb.Append(',');
             }
-            string s = sb.ToString().Trim();
-            return s.Substring(0, s.Length - 1);
+            return sb.ToString(0, sb.Length - 1).Trim();
```
Comment (niksedk) [inline src/Logic/SsaStyle.cs]: Not 100% the same...

**Comment by niksedk** [comment]
Fixed in this commit: https://github.com/SubtitleEdit/subtitleedit/commit/2877320b76893edb9f523f1efa47359fe03adca7

**Pull request closed without merging**